pyproject.toml

```diff
@@ -71,11 +71,27 @@ testpaths = "assesspy/tests"
 legacy_tox_ini = """
 [tox]
 min_version = 4.0
-envlist = py39, py310, py311, py312, py313
+envlist =
+    py{39, 310}-lowest
+    py{310, 311, 312, 313}
+
+[gh-actions]
+python =
+    3.9: py39
+    3.10: py310
+    3.11: py311
+    3.12: py312
+    3.13: py313
```

Comment:
This associates GitHub Action job Python versions with tox envs, such that each version set of tox envs runs in a dedicated GHA job.